tensorflow/lite/experimental/shlo/ops/dot_general.h

```diff
@@ -30,21 +30,21 @@ enum class PrecisionTypes {
 class DotGeneralOp {
  public:
   struct Attributes {
-    absl::Span<int64_t> lhs_batching_dimensions;
-    absl::Span<int64_t> rhs_batching_dimensions;
-    absl::Span<int64_t> lhs_contracting_dimensions;
-    absl::Span<int64_t> rhs_contracting_dimensions;
+    absl::Span<const Axis> lhs_batching_dimensions;
+    absl::Span<const Axis> rhs_batching_dimensions;
+    absl::Span<const Axis> lhs_contracting_dimensions;
+    absl::Span<const Axis> rhs_contracting_dimensions;
```

Comment:
@rascani, can we use Axis type (alias of size_t) here, since that will align with the internal functions like `Shape::Dims`. But it will differ from spec.